examples/timer/index.html

```diff
@@ -1,9 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <title>Yew • Timer</title>
-  </head>
 
-  <body></body>
-</html>
+<head>
+  <meta charset="utf-8" />
+  <title>Yew • Timer</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css" />
```

Comment:
The current styling really doesn't justify pulling in Bulma, does it? It'd be cool if you could style the rest of the page too if we're gonna use Bulma here.

Reply:
Working on this with the latest commits.

Reply:
Removed Bulma in favor of local css.